src/UCTNode.cpp

```diff
@@ -103,8 +103,24 @@ bool UCTNode::create_children(Network & network,
             legal_sum += raw_netlist.policy[i];
         }
     }
-    nodelist.emplace_back(raw_netlist.policy_pass, FastBoard::PASS);
-    legal_sum += raw_netlist.policy_pass;
+
+    // Always try passes if we're not trying to be clever.
+    auto allow_pass = cfg_dumbpass;
+
+    // If we're clever, only try passing if we're winning on the
+    // net score and on the board count.
+    if (!allow_pass && (m_net_eval > 0.8f || nodelist.size() <= 10)) {
```

Comment:
`m_net_eval` is from black's point of view. It needs to be inverted if the player is white.

Reply:
That's what the code above this already does.

Reply:
NN returns value from the point of the current player and the above code makes it black's view for the search. It needs to be inverted back for the current players view for this check.